app/styles/_utils.less

```diff
@@ -16,3 +16,8 @@
 .hide-if-empty:empty {
   display: none !important;
 }
+
+// generic classes not included with twbs
+.inline-block {
+  display:inline-block;
+}
```

Comment:
I changed it from a helper class to a semantic class `.appended-icon` that can be reused.